### DISCUSSION TITLE
errorbars in histograms

### DISCUSSION THREAD

**Discussion by rogerbalsach** [MadAnalysis/madanalysis5#74] [Q&A] [answered]
Dear all,

I don't know if this is the correct place to ask this question, so please feel free to redirect me in case it is necessary.

I'm using MadGraph to generate some data for an e+e- > mu+ mu- gamma process and I now want to use MadAnalysis to plot a histogram of the photon's energy.

Fortunately, with the default card in MadGraph is not difficult to understand how to generate the histogram (I'm using "plot E(a[1]) bin emin emax").
The histogram is fine but I now want to know if there is a way to generate also the error of each bin in the histogram, to be able to compare it with other calculations. I've taken a look at some MA5 documentation but couldn't find a way to do it. 

So, is this something possible?

Thank you

**Comment by jackaraz** [comment, marked as answer]
Hi @Gaussian-art, currently, uncertainties can not be included in the histogramming tool. This feature is in our TODO list.
